src/main/java/racinggame/view/InputView.java

```diff
@@ -1,66 +1,61 @@
 package racinggame.view;
 
-import racinggame.domain.Utils;
+import static racinggame.exception.ExceptionBaseMessage.EXCEEDED_RETRIES_EXCEPTION_MESSAGE;
+
+import java.util.Scanner;
 import racinggame.domain.Validator;
+import racinggame.exception.InputBlankException;
 
 public class InputView {
 
+    public static final String QUIT = "q";
     private static final String SEPARATOR = ",";
     private static final String BLANK = "\\s+";
     private static final String EMPTY_STRING = "";
 
-    private InputView() {}
-
-    public static String[] getCarNames() {
-        String[] carNames;
-
-        while (true) {
-            String inputCarNames = inputCarNames();
-            carNames = splitNameByComma(inputCarNames);
+    private static int tryCount = 1;
 
-            try {
-                Validator.validatePossibleToStart(carNames);
-                break;
-            } catch (IllegalArgumentException e) {
-                System.out.println(e.getMessage());
-            }
-        }
+    private InputView() {}
 
-        return carNames;
+    public static String inputCarNames() {
+        System.out.printf("경주할 자동차 이름을 %s로 구분해서 입력하세요.(%d대 이상)\n", SEPARATOR, Validator.MINIMUM_PLAYER);
+        return getInput();
     }
 
-    private static String inputCarNames() {
-        System.out.println(String.format("경주할 자동차 이름을 '%s'로 구분해서 입력하세요.(%d대 이상)", SEPARATOR, Validator.MINIMUM_PLAYER));
-        return removeBlank(Utils.getInput());
-    }
+    public static String inputTrial() throws IllegalAccessException {
+        System.out.print("시도 횟수를 입력해 주세요: ");
+        String input = removeBlank(getInput());
+
+        try {
+            Validator.checkIsBlank(input);
+            Validator.isNumber(input);
+        } catch (InputBlankException | NumberFormatException e) {
+            System.out.println(e.getMessage());
+            input = retryInputTrial();
+        }
 
-    private static String[] splitNameByComma(String inputCarNames) {
-        return inputCarNames.split(SEPARATOR);
+        return input;
     }
 
-    private static String removeBlank(String trial) {
+    private static String removeBlank(final String trial) {
         return trial.replaceAll(BLANK, EMPTY_STRING);
     }
 
-    public static int getTrial() {
-        String trial;
-
-        while (true) {
-            trial = inputTrial();
-
-            try {
-                Validator.isNumber(trial);
-                break;
-            } catch (IllegalArgumentException e) {
-                System.out.println(e.getMessage());
-            }
+    private static String retryInputTrial() throws IllegalAccessException {
+        if (tryCount > 5) {
+            throw new IllegalAccessException(EXCEEDED_RETRIES_EXCEPTION_MESSAGE);
```

Comment:
```suggestion
            throw new IllegalArgumentException(EXCEEDED_RETRIES_EXCEPTION_MESSAGE);
```
`IllegalArgumentException`가 더 어울리는 표현일 것 같습니다.

참고 - [https://runebook.dev/ko/docs/openjdk/java.base/java/lang/illegalaccessexception](https://runebook.dev/ko/docs/openjdk/java.base/java/lang/illegalaccessexception)

Reply:
감사합니다! 해당 부분 제거하였습니다!